docs/en/ingest-management/commands.asciidoc

```diff
@@ -176,24 +176,26 @@ elastic-agent enroll --fleet-server-es <string>
                      [--fleet-server-cert <string>] <1>
                      [--fleet-server-cert-key <string>]
                      [--fleet-server-es-ca <string>]
+                     [--fleet-server-es-ca-trusted-fingerprint <string>] <2>
```

Comment:
I think this should be shown in the install command, too, right? (I plan to put all the options into one reference topic at some point but haven't had time. For now, we need to duplicate.

Reply:
done